Riot/Managers/Analytics/AnalyticsScreen.swift

```diff
@@ -0,0 +1,114 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import AnalyticsEvents
+
+@objc enum AnalyticsScreen: Int {
```

Comment:
I think we discussed this one: It solves a couple of things:
- `AnalyticsEvent.Screen.ScreenName` can't be bridged into objective-c (it is nested and is backed by Strings).
- We can map locally some screens such as `Favourites` to `MobileFavourites` in the `AnalyticsEvent.Screen.ScreenName` enum

And because of that we get the side effect of decoupling the app from the library.